dashboard/apps/prototype/management/commands/cache.py

```diff
@@ -25,6 +26,7 @@ def generate(self):
         """
         generate cache
         """
+        cache_tools.METHOD_CACHE_WRITE_ONLY = True
```

Comment:
if you are only caching people_costs why don't you call it directly in the cache manage task

i.e. for the entire project:

```python
for for sdate, edate in slice_time_window(project.first_task.start_date, project.last_task.end_date):
    project.people_costs(sdate, edate, ignore_cache=True)

for for sdate, edate in slice_time_window(project.first_task.start_date, date.today()):
    project.people_costs(sdate, edate, ignore_cache=True)

for for sdate, edate in slice_time_window(project.first_task.start_date, project.last_task.end_date, freq='MS'):
    project.people_costs(sdate, edate, ignore_cache=True)
```

Still a bit messy but saves having to set the constant in the module which could cause unexpected behaviour